web/templates/password.html

```diff
@@ -7,7 +7,7 @@ <h2 class="theme-heading">Log in to Your Account</h2>
       <div class="theme-form-label">
         <label for="userid">Username</label>
       </div>
-	  <input tabindex="1" required id="login" name="login" type="text" class="theme-form-input" placeholder="username" {{ if .Username }} value="{{ .Username }}" {{ else }} autofocus {{ end }}/>
+	  <input tabindex="1" required id="username" name="username" type="text" class="theme-form-input" placeholder="username" {{ if .Username }} value="{{ .Username }}" {{ else }} autofocus {{ end }}/>
```

Comment:
do we need this?  I thought you already convert it to json API so we don't need the templates?

Reply:
In fact.. I think front end should be a part of this repo... json API is not a good way